src/tools/merssh/command.cpp

```diff
@@ -182,6 +182,10 @@ QString Command::remotePathMapping(const QString& command) const
 		     (Utils::HostOsInfo::isWindowsHost()?Qt::CaseInsensitive:Qt::CaseSensitive));
     }
     result = result.trimmed();
+
+    qDebug() << "ssh command:" << command;
+    qDebug() << "ssh remote command:" << result;
```

Comment:
Use qCDebug, check the Mer plugin for an example of how/where to declare/define the related stuff. The message could be also more descriptive - it's not clear it is all just about path mapping.

Reply:
This is not intended to be there. Removed.